designer/client/src/components/toolbarSettings/buttons/ActionButton.tsx

```diff
@@ -15,8 +15,22 @@ export function ActionButton({ name, type }: ActionButtonProps): JSX.Element {
     const processName = useSelector(getProcessName);
     const status = useSelector(getProcessState)?.status;
     const customActions = useSelector(getCustomActions);
-
     const action = useMemo(() => customActions.find((a) => a.name === name), [customActions, name]);
 
-    return action ? <CustomActionButton action={action} processName={processName} processStatus={status} type={type} /> : null;
+    // FIXME: This part requires further changes within periodic scenario engine.
```

Comment:
Ok, if I understand correctly, there is no easy way to make it as a separate button as I proposed [here](https://github.com/TouK/nussknacker/pull/6257#discussion_r1650975659), that's why, we need this workaround with a name. Do we have any plan to move it inside core deployment? 

BTW: typo scenearios -> scenarios :)

Reply:
Yes, this will be integrated inside core deployment and scheduling engine.

Typos fixed.